src/Orleans.Transactions/Development/DevelopmentSiloBuilderExtensions.cs

```diff
@@ -1,24 +1,22 @@
 ﻿using Microsoft.Extensions.DependencyInjection;
 using Orleans.Hosting;
-using Orleans.Transactions.Abstractions;
 
 namespace Orleans.Transactions.Development
 {
```

Comment:
you don't have to address this in this PR. I'm just pointing this out: I think all new nugget are going Microsoft.Orleans.XXX namespace, instead of just Orleans

Reply:
We agreed that we'll leave *namespaces* as Orleans.*.

Reply:
I missed that stand up and maybe I got incorrect information. I thought we agreed that all old assembly which use Orleans.* keeps using it for backward-compact reasons. But new assembly use Microsoft.Orleans.*. Is this not the case?

Reply:
>I thought we agreed that all old assembly which use Orleans.* keeps using it for backward-compact reasons. But new assembly use Microsoft.Orleans.*. Is this not the case?

No, that's not the case. We agreed to stay with Orleans.* namespaces, and revert newly added ones to that convention, too.